azure-storage/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-stream-azure-storage-parent</artifactId>
+	<packaging>pom</packaging>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<artifactId>spring-cloud-stream-app-starters</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<modules>
+		<module>spring-cloud-starter-stream-sink-wasb</module>
```

Comment:
The minimal working stuff is already here - I've done some basic testing, although I'm not certain how to properly implement a true "test", since this depends on active Azure credentials, unless there is some way to mock the functionality of the Azure service itself in an offline way.

Let's reach steady-state on all of your suggested changes and I'll re-open against another branch.